### ISSUE TITLE
training batchsize 

### ISSUE THREAD

**Issue by syusukee** [greatlog/SWAHR-HumanPose#5]
hello, how many gpus did you used? I found that your batchsize maybe is different from the original HigherHRNet, can you share the batchsize you used for training? Thanks a lot!

**Comment by greatlog** [comment]
We use 8 Tesla V100 GPUs. For different models, the number of images per GPU is denoted in the [yml file](https://github.com/greatlog/SWAHR-HumanPose/blob/755582699faeafb89e4e00515677e0d581366012/experiments/coco/higher_hrnet/w32_512_adam_lr1e-3.yaml#L120). The total batch size is the number of  GPUs times the number of images per GPU.

**Comment by syusukee** [comment]
Thanks for your reply! So why not use the origin learning rate and batchsize, did you have tried different settings and chosen the best one?

**Comment by greatlog** [comment]
We cannot get the reported results of Higher HRNet on the original repo, maybe because of the difference of machine. So we try different learning strategies and chose the one that can reimplement Higher HRNet.
